Traceback (most recent call last):
  File "/home/user/eric123/my_project/test/test_load_data.py", line 10, in <module>
    with open('config.yaml', 'r') as f:
FileNotFoundError: [Errno 2] No such file or directory: 'config.yaml'
srun: error: gpu06: task 0: Exited with exit code 1
